src/rust/src/rlib.rs

```diff
@@ -59,33 +59,40 @@ pub fn hor_concat_df(dfs: &VecDataFrame) -> List {
 #[extendr]
 fn min_exprs(exprs: &ProtoExprArray) -> Expr {
     let exprs = exprs.to_vec("select");
-    polars::lazy::dsl::min_exprs(exprs).into()
+    pl::min_exprs(exprs).into()
 }
 
 #[extendr]
 fn max_exprs(exprs: &ProtoExprArray) -> Expr {
     let exprs = exprs.to_vec("select");
-    polars::lazy::dsl::max_exprs(exprs).into()
+    pl::max_exprs(exprs).into()
 }
 
 #[extendr]
 fn coalesce_exprs(exprs: &ProtoExprArray) -> Expr {
     let exprs: Vec<pl::Expr> = exprs.to_vec("select");
-    polars::lazy::dsl::coalesce(exprs.as_slice()).into()
+    pl::coalesce(exprs.as_slice()).into()
 }
 
 #[extendr]
 fn sum_exprs(exprs: &ProtoExprArray) -> Expr {
     let exprs = exprs.to_vec("select");
-    polars::lazy::dsl::sum_exprs(exprs).into()
+    pl::sum_exprs(exprs).into()
 }
 
 #[extendr]
 fn concat_list(exprs: &ProtoExprArray) -> Result<Expr, String> {
     let exprs = exprs.to_vec("select");
-    Ok(Expr(
-        polars::lazy::dsl::concat_list(exprs).map_err(|err| err.to_string())?,
-    ))
+    Ok(Expr(pl::concat_list(exprs).map_err(|err| err.to_string())?))
+}
+
+#[extendr]
+fn concat_str(dotdotdot: Robj, separator: Robj) -> RResult<Expr> {
+    Ok(pl::concat_str(
+        robj_to!(Vec, PLExprCol, dotdotdot)?,
+        robj_to!(str, separator)?,
+    )
+    .into())
```

Comment:
Also it might be better to avoid directly passing `...` from R to Rust (just my personal preference). Maybe consider giving the parameter a name and avoid the `replace` in `robj_to!`, and properly check the content in `...` on the R side.

Reply:
Wait does this actually compiles? The type of the expression seems to mismatch the return type from my perspective.

Reply:
Oops I read the wrong function in the documentation. This is actually correct. Just disregard my comments above.